packages/firebase_admob/lib/firebase_admob.dart

```diff
@@ -406,6 +406,21 @@ class RewardedVideoAd {
       'targetingInfo': targetingInfo?.toJson(),
     });
   }
+
+  /// Sets the user id to be used in server-to-server reward callbacks
+  Future<bool> setUserId({@required String userId}) {
```

Comment:
We don't typically use setter methods in dart. The [style guide](https://dart.dev/guides/language/effective-dart/design#do-use-setters-for-operations-that-conceptually-change-properties) recommends using accessors e.g.

```dart
String _userId;
bool get userid => _userId;
set userId(String userId) { 
   _invokeBooleanMethod("setRewardedVideoAdUserId", <String, dynamic>{
     'userId': userId,
   });
}
```
And `MethodCall`s also run sequentially, so it would finish before any other would be called.